iroh-resolver/src/resolver.rs

```diff
@@ -351,14 +271,17 @@ impl Out {
     /// is unixfs
     pub fn named_links(&self) -> Result<Vec<(Option<&str>, Cid)>> {
         match &self.content {
-            OutContent::Unixfs(node) => node.links().map(|l| l.map(|l| (l.name, l.cid))).collect(),
+            // TODO(ramfox): add back in when we figure out circular dependencies
+            // OutContent::Unixfs(node) => node.links().map(|l| l.map(|l| (l.name, l.cid))).collect(),
+            OutContent::Unixfs(_) => todo!(),
             _ => {
                 let links = self.content.links();
                 links.map(|l| l.into_iter().map(|l| (None, l)).collect())
             }
         }
     }
 
+    // TODO(ramfox): figure out circular dependencies
```

Comment:
what circular dep is still here?

Reply:
none, left over note to myself

thanks for catching